### ISSUE TITLE
_C. auris_ specific orthogroups

### ISSUE THREAD

**Issue by hezhaobin** [binhe-lab/C037-Cand-auris-adhesin#6]
I looked into this issue raised by @janfassler. My analysis and discussion can be read [here](https://htmlpreview.github.io/?https://github.com/binhe-lab/C037-Cand-auris-adhesin/blob/master/01-global-adhesin-prediction/analysis/protein-family-classification/OrthoMCL-4sps.nb.html#c_auris_specific_orthogroups). Briefly, these are what I found:
1. The predicted adhesins in _C. auris_ that belongs to a _C. auris_ specific orthogroup (i.e. none of the three other species have members in it) DO NOT have lower FungalRV scores than the rest of the predicted proteins. This observation is also true for orthogroups specific to the other species.
1. Manually checking one of the 7 _C. auris_ specific orthogroups revealed two types of cases:
    1) there ARE orthologs in _C. albicans_, and they are actually annotated as "GPI-anchor" "cell wall proteins", but both FungalRV and FaaPred predicted them to be non-adhesins.
    1) the _C. albicans_ ortholog is just below the 0.511 cutoff and would have been included if I looked at the FaaPred subset (ongoing analysis).

For the first case above, it is not immediately clear to me whether that represents a false negative by the predictor for the _C. albicans_ proteins, or a false positive for the _C. auris_. For the latter, I could look at the FaaPred subset and reevaluate the conclusion about the distribution of orthogroups shared by 1, 2, 3 and 4 species.

More details in the link above.

**Comment by janfassler** [comment]
I'm posting a draft NJ tree (midpoint rooted) that I constructed using the N terminal ~300 aa of C. auris adhesins similar to Lindsey's (top branch, blue, labeled LINDSEY).  You'll see that the tree also contains Rachel's protein (green, labeled RACHEL). Also, all the C. albicans protein "relatives" fall out into an unrelated clade.  I intend to refine this tree through multiple iterations with added sequences and to ask whether the members of each of the three distinct auris clades are related by their C terminal repeat types or their protein domain architecture.  I also plan to dig deeper into the amyloid propensity of these proteins.  I think we could plan to discuss these topics next Monday (a week from today) if everyone is available. 
[NJTree_LS&RS_proteins.docx](https://github.com/binhe-lab/C037-Cand-auris-adhesin/files/4781114/NJTree_LS.RS_proteins.docx)

**Comment by hezhaobin** [comment]
Yes I am available! I'll also check @lindseyfaye and @rsmoak's proteins in my orthomcl results.

**Comment by lindseyfaye** [comment]
Next Monday works for me as well.

**Comment by rsmoak** [comment]
That works for me as well. I'm free all day.
